pysat/instruments/pysat_testing_xarray.py

```diff
@@ -95,34 +122,35 @@ def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
     # need to create simple orbits here. Have start of first orbit
     # at 2009,1, 0 UT. 14.84 orbits per day
     time_delta = date - root_date
-    mlt = test.generate_fake_data(time_delta.total_seconds(), num_array,
-                                  period=5820, data_range=[0.0, 24.0])
+    mlt = mm_test.generate_fake_data(time_delta.total_seconds(), uts,
+                                     period=5820, data_range=[0.0, 24.0])
```

Comment:
Period and data range are specified specifically a lot.  It would be better to make these and other such variables defined at a higher level and called in each of these instances.  That way, when we decide to change things it will be easier to maintain.

Reply:
added routines in the methods file to specify which periods and ranges to use so that this is consistent across files.